entities.csv

```diff
@@ -409,16 +421,21 @@ MAC
 MAC_1
 Macrophage_inflammatory_proteins
 MAF
+MAP

```

Comment:
Since we use names as synonyms in some places, I'm mildly concerned about this causing NER issues when integrated with readers (like if some variant of "MAP kinase" is not recognized, it could be cropped to MAP and mapped to this entry). Renaming to "Microtubule_associated_proteins" or something like that might be safer.

Reply:
If possible I think we should make an exception to the use of names as synonyms for Famplex. There are many existing entries that could cause similar problems. That said, I 100% agree that  "Microtubule_associated_proteins" is a better name for this family

Reply:
I agree, it would be better to create names explicitly. We'd just have to go through and do it systematically. This PR https://github.com/sorgerlab/famplex/pull/87 and issue https://github.com/sorgerlab/famplex/issues/142 are also relevant.